Mathlib/Logic/Basic.lean

```diff
@@ -959,6 +959,9 @@ theorem ExistsUnique.unique₂ {α : Sort*} {p : α → Sort*} [∀ x, Subsingle
   exact h.unique ⟨hpy₁, hqy₁⟩ ⟨hpy₂, hqy₂⟩
 #align exists_unique.unique2 ExistsUnique.unique₂
 
+@[simp] lemma elim_exists_prop (q : Prop → Prop) : (∃ p, q p ∧ p) ↔ q True := by
```

Comment:
I think a better lemma would be
```suggestion
@[simp] lemma Prop.exists (q : Prop → Prop) : (∃ p, q p) ↔ q True ∨ q False := by
```

Reply:
This already exists (no pun intended), under `Prop.exists_iff`. I will remove this new simp lemma from the file and instead give a `Prop.exists_iff` hint to the two `simp`s in FrameAdjunction that need it. Thanks!